index.html

```diff
@@ -75,18 +76,163 @@ <h1>ADEMOLA OWOEYE</h1>
                 <i class="fa-brands fa-twitter"></i></li>
                 <li class="M-icon">
                 <i class="fa-solid fa-m"></i></li>
-                </div>
+                </ul>
+            </div>
 
-               <div class="work-down-arrow">
+                <div class="work-down-arrow"> 
                 <div class="icon">
                     <i class="fa-solid fa-arrow-down"></i>
                     </div>
-                <div class="work"> work </div>
-              </div>
+                   <div class="wor" > work </div> 
+                 </div> 
+    </div>            
+</section>
+        
 
-            </div>
 
+
+<section class="work">
```

Comment:
- [ ] KIndly ensure that your design matches the [Figma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A2346) chosen 👍. Kindly pay more attention to your padding, margins, and colors. 

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/177009817-d1fcc900-c1db-443a-a960-e000a0ffbb27.png) | ![image](https://user-images.githubusercontent.com/63473584/177009770-b1bb66f5-f816-431b-b289-23bf9105d20a.png)  |